tests/installation/partitioning/modify_suggestion_encrypt_partition.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
```

Comment:
for me it sounds more natural, 'modify_proposal_encrypt_partition, because 'modify_suggested_partitioning_encrypt_partition' is too long. But it is just a suggestion, leave it as it is if you still like more the name you chose.

Reply:
Here as well, I used "Suggestion" because page's header is "Suggested Partitioning" but as you mentioned, a bit too long. I prefer to keep "Suggestion" to "Proposal" because they both make sense in English but "Suggestion" is closer to "Suggested Partitioning".

Reply:
After sleeping on it, I renamed it to "modify_suggested_partitioning_encrypt". WDYT?

Reply:
As we already have `partitioning` in the name, I would propose `modify_proposal_encrypt` or `edit_proposal_encrypt`. In general we also could even make it atomic, so it's purpose is 100% clear and name it `edit_proposal_encrypt_home`. That would also give us flexibility as we want to encrypt `home` partition and not `vda3` as it might correspond to something else.

Reply:
ok, if two out of three prefer "proposal" to "suggestion" there must be a reason for that.